assets/tests/CodeCheckerService.js

```diff
@@ -25,10 +25,18 @@ tie.factory('CodeCheckerService', [
       return code.indexOf(functionDefinition) !== -1;
     };
 
+    var _checkIfClassDefinitionExistsInCode = function(className, code) {
+      var classDefinition = 'class ' + className + '(object):';
```

Comment:
So this function may check both the old/new style class definition right? (I'm a little confused by why you bold @classmethod).